### ISSUE TITLE
Trouble running Hello World example

### ISSUE THREAD

**Issue by DangItJason** [gologinapp/gologin#59]
Versions:
 Node.js v16.15.1
 gologin@1.0.50
 puppeteer-core@14.2.1

Hey! I'm having issues running the hello world example (did not change a single line). The following is my console output after running ` DEBUG=gologin* node main.js`. I copy and pasted the token and the profile_id from the gologin desktop client. The profile created is all default. 

  gologin INIT GOLOGIN 629e9cf07e3c956a7a3152a9 +0ms
(node:251) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
(Use `node --trace-deprecation ...` to show where the warning was created)
Orbita downloading [==============================] 32/mps 126.57/126.57MB 100% 0.0s

Download completed
Checking Orbita archive
Extracting Orbita
Trace: Error: Command failed: tar xzf /home/adameve62/.gologin/browser/orbita-browser-latest.tar.gz --directory /home/adameve62/.gologin/browser/extracted-browser
/usr/bin/gzip: 1: ELF: not found
/usr/bin/gzip: 3: : not found
/usr/bin/gzip: 4: Syntax error: "(" unexpected
tar: Child returned status 2
tar: Error is not recoverable: exiting now

    at ChildProcess.exithandler (node:child_process:398:12)
    at ChildProcess.emit (node:events:527:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Socket.<anonymous> (node:internal/child_process:451:11)
    at Socket.emit (node:events:527:28)
    at Pipe.<anonymous> (node:net:709:12) {
  code: 2,
  killed: false,
  signal: null,
  cmd: 'tar xzf /home/adameve62/.gologin/browser/orbita-browser-latest.tar.gz --directory /home/adameve62/.gologin/browser/extracted-browser',
  stdout: '',
  stderr: '/usr/bin/gzip: 1: \x7FELF\x02\x01\x01\x03: not found\n' +
    '/usr/bin/gzip: 3: \x03\x03\x01: not found\n' +
    '/usr/bin/gzip: 4: Syntax error: "(" unexpected\n' +
    'tar: Child returned status 2\n' +
    'tar: Error is not recoverable: exiting now\n'
}
    at /mnt/c/Users/nguye/Projects/UberBot/main.js:11:15
    at async /mnt/c/Users/nguye/Projects/UberBot/main.js:10:31
Invalid status

Not really sure what to make of this! Has anyone ran into a similar issue?

**Comment by alexandivv** [comment]
Do `npm i gologin`, **wait for the installation to finish**

Install puppeteer-core `npm i puppeteer-core`

Copy example
```
const puppeteer = require('puppeteer-core');
const GoLogin = require('gologin');

(async () =>{
    const GL = new GoLogin({
        token: 'yU0token',
        profile_id: 'yU0Pr0f1leiD',
    });

    const { status, wsUrl } = await GL.start().catch((e) => {
      console.trace(e);
      return { status: 'failure' };
    });

    if (status !== 'success') {
      console.log('Invalid status');
      return;
    }

    const browser = await puppeteer.connect({
        browserWSEndpoint: wsUrl.toString(), 
        ignoreHTTPSErrors: true,
    });

    const page = await browser.newPage();
    await page.goto('https://myip.link/mini');   
    console.log(await page.content());
    await browser.close();
    await GL.stop();
})();
```
Replace yU0token with your token

Replace yU0Pr0f1leiD with your profile id

Run `DEBUG=gologin* node example.js`

**Comment by DangItJason** [comment]
It's refusing to connect, the following is part of my error output:

Trace: Error: connect ECONNREFUSED 127.0.0.1:26792
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1187:16) {
  errno: -111,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 26792,
  attempts: 5
}

**Comment by swherden** [comment]
Have the following error when running example: 

```sh
Error: Exactly one of browserWSEndpoint, browserURL or transport must be passed to puppeteer.connect
    at assert (/Users/username/Documents/GitHub/tiktok-replicator/node_modules/puppeteer-core/lib/cjs/puppeteer/util/assert.js:28:15)
    at _connectToCDPBrowser (/Users/username/Documents/GitHub/tiktok-replicator/node_modules/puppeteer-core/lib/cjs/puppeteer/common/BrowserConnector.js:63:28)
    at PuppeteerNode.connect (/Users/username/Documents/GitHub/tiktok-replicator/node_modules/puppeteer-core/lib/cjs/puppeteer/common/Puppeteer.js:79:63)
    at PuppeteerNode.connect (/Users/username/Documents/GitHub/tiktok-replicator/node_modules/puppeteer-core/lib/cjs/puppeteer/node/PuppeteerNode.js:113:22)
    at /Users/username/Documents/GitHub/tiktok-replicator/src/test.js:22:37
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by swherden** [comment]
In my case, I had to add 
```
waitWebsocket: true
```

to my params in constructor

**Comment by stepjet** [comment]
@swherden thank you for the feedback. this issue was fixed in 1.0.58 version of gologin npm package
